aws/aws-ecs/nodejs-terraform/waypoint.hcl

```diff
@@ -0,0 +1,33 @@
+project = "njs-ecs-tfc"
+
+app "njs-ecs-tfc" {
+  build {
+    use "pack" {}
+    registry {
+      use "aws-ecr" {
+        region     = "us-west-2"
+        repository = "njs-ecs-tfc"
+        tag        = "latest"
+      }
+    }
+  }
+
+  deploy {
+    use "aws-ecs" {
+      region = "us-west-2"
+      memory = "512"
+      cluster = var.ecs_cluster
+    }
+  }
+}
+
+variable "ecs_cluster" {
+  default = dynamic("terraform-cloud", {
+    organization = "hc-waypoint"
+    workspace    = "hashiconf-demo"
```

Comment:
Since this is in the waypoint examples repo, I wonder if this should be more generic here?